module/settings.js

```diff
@@ -34,4 +40,14 @@ export const registerSystemSettings = function () {
     type: Boolean,
     default: false,
   });
+
+  game.settings.register("yzecoriolis", "Initiative", {
+    name: game.i18n.localize("YZECORIOLIS.SettingInitiative"),
+    hint: game.i18n.localize("YZECORIOLIS.SettingInitiativeHint"),
+    scope: "world",
+    config: true,
+    type: Boolean,
+    default: false,
+    onChange: debouncedReload,
```

Comment:
Instead of reloading the window, could we just reasign `CONFIG.Combat.initiative`. Maybe extract a function with the code in `yzecoriolis.js` below and call that function on `Hooks.once("init",...)` and `onChange`?

Reply:
Maybe but I'm not sure if this will work out and I don't have an useful idea how to implement this.

I oriented myself with the Foundry Docs and that was the way they described it there. https://foundryvtt.wiki/en/development/guides/handling-data

But if you can make it work, I would gladly accept it. :)

Reply:
Thx! Is refactored with your changes